### DISCUSSION TITLE
evolution of Bionomia (total numbers)

### DISCUSSION THREAD

**Discussion by abubelinha** [bionomia/bionomia#375] [General]
Hi. In order to be more convincent when I talk somebody about this wonderful project, I wonder if there is any easy way of seeing Bionomia's total numbers along time.
i.e. number of specimens attributed, number of attributions, number of different people attributed, number of scribes ...
It would be something nice to see. I'd expect it to show many growing lines but who knows

Maybe you have already published a previous report like that somewhere?
Sorry if it is already linked but I am not being smart enough to find something similar.
If not, I think it would be nice to see some graphical reports in the website.

But in the mean time -to put it easier and quicker- a downloadable .csv (or a Zenodo Dshorthouse citable DOI) would do it.
This would be a quite small file (and hopefully easy to do). A possible header:
`year, month, scribes_count, attributions_count, distinct_recorders_attributed, distinct_identifiers_attributed,  distinct_attributed_GBIF_records, total_GBIF_records_available`

(if you like the idea and want to go beyond, just complicate it a bit more aggregating by Kingdoms and/or continents ... although I'd be happier without that)

If nothing of this is possible, thanks anyway !!
@abubelinha

**Comment by dshorthouse** [comment]
Have a look at https://bionomia.net/statistics, which is linked from the footer under "About"/"Statistics". I think what you were looking for is all here, but happy to add more provided the data can support it.

**Comment by abubelinha** [comment]
Indeed that was it. Sorry about not finding it (again).
I think I had already seen it before but I wasn't sure it was here or in some paper (or just my imagination).
